harvester/harvest.py

```diff
@@ -57,6 +58,126 @@
 
 ROOT_DIR = Path(__file__).parents[1]
 
+# harvest worker count
+harvest_worker_sync_count = int(os.getenv("HARVEST_WORKER_SYNC_COUNT"))
```

Comment:
Suggestion (non-blocking): If the env variable is missing, this will give an error I think. Can we use a safe default as the second argument to getenv?
```suggestion
harvest_worker_sync_count = int(os.getenv("HARVEST_WORKER_SYNC_COUNT", 1))
```

Reply:
Thank you!